flang/runtime/CMakeLists.txt

```diff
@@ -281,3 +281,26 @@ add_flang_library(FortranRuntime
 
   INSTALL_WITH_TOOLCHAIN
 )
+
+if (DEFINED MSVC)
+  set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreaded)
+  add_flang_library(FortranRuntime.static ${sources}
```

Comment:
When targeting the MSVC ABI, the plain `FortranRuntime` library added above should be excluded.  Only the per-CRT variants should exist, because choosing a CRT variant is not optional.

Reply:
The plain FortranRuntime library is linked by the Runtime tests, which need to be built against whatever the user built LLVM with, which we can't necessarily find out that easily I think. It should not have INSTALL_WITH_TOOLCHAIN set though, I'll remove that